packages/product-service/addProduct/addProduct.ts

```diff
@@ -0,0 +1,67 @@
+import { APIGatewayProxyHandler } from 'aws-lambda';
+import { Client } from 'pg';
+import { Product } from '../models';
+import { cors } from '../cors';
+
+export const addProduct: APIGatewayProxyHandler = async (event) => {
+  console.log(`Request: ${JSON.stringify(event, null, 2)}`);
+
+  if (!event.body) {
+    return {
```

Comment:
@dkovalishyn 

This request:

```
{
    "title": "Task 4 review postman test",
    "price": 10.0
}
```

will throw 500 error, because the `count` field should be `NOT NULL`, according to your DB schema.

Please, add a simple validation for required fields, so the aforementioned request will return a `400` error.